src/fides/api/schemas/dataset.py

```diff
@@ -36,17 +38,17 @@ def valid_data_categories(
             return _valid_data_categories(v, defined_data_categories)
 
     class FieldDataCategoryValidation(DatasetField, DataCategoryValidationMixin):
-        fields: Optional[List["FieldDataCategoryValidation"]]
+        fields: Optional[List["FieldDataCategoryValidation"]]  # type: ignore[assignment]
 
     FieldDataCategoryValidation.update_forward_refs()
 
     class CollectionDataCategoryValidation(
         DatasetCollection, DataCategoryValidationMixin
     ):
-        fields: List[FieldDataCategoryValidation] = []
+        fields: Sequence[FieldDataCategoryValidation] = []  # type: ignore[assignment]
```

Comment:
Where is this coming from?

Reply:
This is required due to Lists in Python being invariant, as a result of their mutability